### ISSUE TITLE
Nested Subcommands Are Broken on v1.4.1 

### ISSUE THREAD

**Issue by nisdas** [prysmaticlabs/prysm#9235]
# 🐞 Bug Report

### Description

Nested Subcommands are Broken on `v1.4.1` . Any subcommand whose parent was a subcommand 
is affected by this bug.  #9129 introduced stricter validation of commands and arguments which did end up 
breaking this command flow.

### Has this worked before in a previous version?

Yes this has been working previously till v1.3.11 .

## 🔬 Minimal Reproduction

```
./validator accounts list
```

## 🔥 Error

```
[2021-07-21 12:43:04] ERROR main: unrecognized argument: list
```

**Comment by prestonvanloon** [comment]
# :bulb: Workaround

There are no breaking changes between v1.3.11 (previous release) and v1.4.1. Therefore, it is possible and safe to run validator account commands with the prior version until a release for v1.4.2 is announced in the coming days.

If using the prysm.sh script, set the environment variable `USE_PRYSM_VERSION` to `v1.3.11` and run your commands.
```
USE_PRYSM_VERSION=v1.3.11 ./prysm.sh validator accounts list

USE_PRYSM_VERSION=v1.3.11 ./prysm.sh validator accounts import
```

If using docker `:stable` image, use the tag `:v1.3.11` for any _validator_ subcommands as pointed out above.

Lastly, it is safe to run v1.3.11 version of validator with the London hardfork ready beacon-chain at version v1.4.1. 
If validator subcommands are blocking staking operations, it may be desired to run the prior version of the validator until v1.4.2 is released.

## TL;DR
It's safe to run the validator at version v1.3.11 and the beacon node at version v1.4.1. Expect a patch release in 24 hours.

**Comment by prestonvanloon** [comment]
This is fixed in [v1.4.2](https://github.com/prysmaticlabs/prysm/releases/tag/v1.4.2)
